README.md

```diff
@@ -200,6 +200,51 @@ import { renderRichText } from "@storyblok/js";
 const renderedRichText = renderRichText(blok.richtext);
 ```
 
+You can set a **custom Schema and component resolver globally** at init time by using the `richText` init option:
+
+```js
+import { richTextSchema, storyblokInit } from "@storyblok/js";
```

Comment:
Isn't the name we need to use here is `RichTextSchema` and not `richTextSchema`?

Reply:
Actually it's camelcase as it's an object, not a class. Standard JavaScript conventions :)

Reply:
@alexjoverm I mean that in the way it's implemented it won't work for me as `richTextSchema`, I had to capitalise it